MCprep_addon/mcprep_ui.py

```diff
@@ -314,14 +318,135 @@ def feature_set_update(self, context):
 	tracking.Tracker.feature_set = self.feature_set
 	tracking.trackUsage("feature_set", param=self.feature_set)
 
+def get_unsimple_list(mcprep_add, skins, world_tools, spawner):
+	classes = []
+	if mcprep_add:
+		classes.append(MCPREP_MT_3dview_add)
+	if skins:
+		classes.append(MCPREP_PT_skins)
+	if world_tools:
+		classes.append(MCPREP_PT_world_tools)
+	if spawner:
+		spawn_classes = (
+			MCPREP_PT_spawn, 
+			MCPREP_PT_mob_spawner, 
+			MCPREP_PT_item_spawner,
+			MCPREP_PT_model_spawner,
+			MCPREP_PT_entity_spawner,
+			MCPREP_PT_effects_spawner,
+			MCPREP_PT_meshswap_spawner,
+		)
+		classes = classes + [cls for cls in spawn_classes]
+	return classes
+
+def simple_class_reregister(mcprep_add, skins, world_tools, spawner):
```

Comment:
Can you add a screenshot of what this UI looks like and high level explain the differences made by these simple variants? I see the [release here](https://github.com/StandingPadAnimations/MCprep-Kaion/releases/tag/3.4.1.1), but not sure what it looks like in the broader context

At a high level, I'm weary of registering and unregistering classes midway through script execution, it's a bit abnormal. I'm much more okay with dynamically deciding to draw different already-registered UI elements based on preferences options. The memory overhead is negligent to have both and conditionally draw.